### DISCUSSION TITLE
quantity on cart drawer getting hidden

### DISCUSSION THREAD

**Discussion by arkodeepg** [EvgeniyMukhamedjanov/liquid-ajax-cart#90] [Q&A]
issue flow 
add the product -> click on add gift wrap

There is a logic running in the background to check when the button is click then liquid ajax add happens 
testing link - https://nu924cjyt573ecuq-26900106.shopifypreview.com/products/grey-lavender-22-piece-dinner-set-for-6

also the atc and quantity events are a bit slower, is there something I can do to make it faster?

**Comment by EvgeniyMukhamedjanov** [comment]
Hi @arkodeepg 
I noticed your input field doesn't have value in the html:
<img width="338" alt="image" src="https://github.com/EvgeniyMukhamedjanov/liquid-ajax-cart/assets/13817355/112fb647-62c0-4331-84b1-623b223a983a">
Can you make sure that you pass the value in your section liquid file. So it should be something like this:
```liquid
<input data-ajax-cart-quantity-input="{{ line_item_index }}"
  name="updates[]" 
  <!-- line item quantity should be passed into the value:  -->
  value="{{ line_item.quantity }}"  
  type="number" 
  form="my-ajax-cart-form" />
```

**Reply by arkodeepg** [reply]
thank you, corrected it. Had it set to {{item.quantity}} hence was not getting the data. Is there any way I can make the cart drawer a bit fater, also there is a small jitter(image gets removed and again added) when quantity is increased ?

**Reply by EvgeniyMukhamedjanov** [reply]
Usually images might flicker when your google chrome dev tools are open, because you might have flag "Disable cache" to be on. Can you test that without the google chrome dev tools, it should be fine.
In terms of speed - we can't make shopify ajax requests work faster. But once you push the theme live, it should work a bit faster. Also can you share the code for the logic which works slow in your opinion. Maybe I could suggest the way how to decrease amount of requests in order to make it faster?
